### ISSUE TITLE
Version 2 will be released soon.

### ISSUE THREAD

**Issue by Edzelf** [Edzelf/ESP32-Radio#474]
I am planning to release a new version that can optionally use a simple PCM5102A dac (I2S) instead of the VS1053 board.  MP3 is already functioning.  I am working on the AAC and maybe Bluetooth A2D.

**Comment by NonaSuomy** [comment]
Will it be much to port this updated I2S code base to the WM8978? https://github.com/Edzelf/ESP32-Radio/issues/110#issuecomment-783481645 with this library https://github.com/CelliesProjects/wm8978-esp32 would be nice as a lot of people have this T-Audio 1.6 board http://www.lilygo.cn/prod_view.aspx?TypeId=50063&Id=1171  https://www.aliexpress.com/item/32846510254.html

**Comment by Ne3Mx** [comment]
I was just about to ask if it could be done without the vs1053 as I'm busy with a PCB for this project. I think I'll hold off on the PCB until V2 is released. The PCB mentioned in the documentation is no longer downloadable.

It will be really great if we can use the i2s and hook it up directly to a class d amp.  Obviously you can still hookup a headphone DAC via i2s.

I was thinking of using a shape similar to a single DIN used for car radios back in the day. The idea will be to have a wide rectangle fave shape that will allow me to hook it up under my kitchen cupboard thereby saving some space. The unit won't have any built in speakers but will use external speakers. It'll also have a 3d printable case for mounting under cupboards.

**Comment by utw3v0** [comment]
> I am planning to release a new version that can optionally use a simple PCM5102A dac (I2S) instead of the VS1053 board. MP3 is already functioning. I am working on the AAC and maybe Bluetooth A2D.

I'm looking forward

**Comment by Tony763** [comment]
Hi, could v2 also make use of ESP32 with PSRAM to increase buffers?

**Comment by Edzelf** [comment]
I don't think that using extra PSRAM for buffering has any positive effect, but if you can prove it, I will.

**Comment by NonaSuomy** [comment]
Can you add an NTP based time server so it can display the time and play a station on an alarm?

**Comment by NonaSuomy** [comment]
You can set an alarm in version 1? nice can you do it through the webui?

**Comment by swanepoeljan** [comment]
> 
> 
> I am planning to release a new version that can optionally use a simple PCM5102A dac (I2S) instead of the VS1053 board. MP3 is already functioning. I am working on the AAC and maybe Bluetooth A2D.

I am new to all this and just curious what advantages this brings?
From my understanding the VS1053 handles decoding while the PCM5102A is just a DAC so decoding would need to be done on the ESP32?

**Comment by Edzelf** [comment]
Right.  The VS1053 will be the best choice.  But the ESP32 will be able to do the decoding with some limitations.

**Comment by marcdraco** [comment]
Bluetooth?


SWOOOOON!

**Comment by B4stl3r** [comment]
Hi Ed,

check this out:
https://www.berrybase.de/neu/raspiaudio-muse-esp32-proto?c=2473
https://github.com/RASPIAUDIO/museProto_radio

**Comment by alfrede** [comment]
Hi Ed, can i help you with making a pcb? i should need a schematic and would design it with kicad, so everybody can make it's own pcb.

**Comment by Edzelf** [comment]
@alfrede: Contact Ne3Mx.  See the 3rd item in this thread.

**Comment by NonaSuomy** [comment]
How's the progress going?

**Comment by Edzelf** [comment]
An experimental version is released today.

**Comment by NonaSuomy** [comment]
Great work! Is it on a different branch to grab?

**Comment by NonaSuomy** [comment]
See it here https://github.com/Edzelf/ESP32Radio-V2
